doc/aurchain.md

```diff
@@ -0,0 +1,23 @@
+# aurchain
+
+## Name
+
+aurchain - Solve and order dependencies using the AUR RPC
+
+## Synopsis
+
+aurchain package [package, ...]
+
+## Description
+
+aurchain takes a list of package names and then solves and orders the dependencies for those packages.
+
+## Operations
+
+* __-v__ Output more
```

Comment:
Then it is probably best to remove it from the documentation as well (the switch is still in the code though).